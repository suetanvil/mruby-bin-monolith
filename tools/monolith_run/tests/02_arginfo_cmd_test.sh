#!/bin/bash

set -e

../mrb_run print_arginfo.mrb a b c
