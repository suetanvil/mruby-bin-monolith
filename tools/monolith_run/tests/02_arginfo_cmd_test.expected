$0: print_arginfo.mrb
ARGV[0]: a
ARGV[1]: b
ARGV[2]: c
MC::ExePath: $ROOT/mrb_run
MC::ProgPath: print_arginfo.mrb
MC::IsConstructed: false
Done.
